packages/components/src/SearchInput.tsx

```diff
@@ -10,12 +10,13 @@ import { ContextActions } from './context-actions';
 interface SearchInputProps {
   value: string;
   placeholder: string;
+  endPlaceholder?: string; // Optional placeholder text shown on the right side of the input when empty
```

Comment:
Two things:
1. The Doc string needs to go above the prop rather than in line, and also use `/**  */` instead of `//`, or else it does not appear in the autocomplete/docs. Right now if you try and use this prop:
<img width="713" height="133" alt="image" src="https://github.com/user-attachments/assets/ab7b6500-42ed-4e5a-8ed1-b639faa763ef" />
vs. when you put the comment above and use `/**  */` style comment:
<img width="723" height="171" alt="image" src="https://github.com/user-attachments/assets/8469dc06-71c3-4409-81fa-b937d91f62f2" />
2. "Optional" wording is redundant here; it's optional as indicated by the `?` already. I'd just get rid of that. (This is a bit of a nitpick and really doesn't matter).
```suggestion
  /** Placeholder text shown on the right side of the input when empty */
  endPlaceholder?: string;
```